modules/rostests/apitests/shell32/ShellExecCmdLine.cpp

```diff
@@ -13,7 +13,8 @@
 #include <debug.h>
 #include <stdio.h>
 
-//#define ShellExecCmdLine ShellExecCmdLine
+#define ShellExecCmdLine ShellExecCmdLine // In testing on ReactOS
+//#undef ShellExecCmdLine // In testing on Vista+
```

Comment:
?

Reply:
ShellExecCmdLine is Vista+ private function. So WinXP doesn't export it. Defining ShellExecCmdLine macro will enable the below synced code. We can't automatically switch the behavior.

Reply:
If you completely remove the define,
rename the function (inside the apitest) to `proxy_ShellExecCmdLine` (or something along those lines),
then if the GetProcAddress failes, you fill the `g_pShellExecCmdLine` with the `proxy_ShellExecCmdLine` instead of the actual function, it should work for both?
unless I am missing something

Reply:
OK. Done.